[2025-07-30T03:12:22.617+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-30T03:12:22.757+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ppppppppppppppppp.load_data_task scheduled__2025-06-25T00:00:00+00:00 [queued]>
[2025-07-30T03:12:22.765+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ppppppppppppppppp.load_data_task scheduled__2025-06-25T00:00:00+00:00 [queued]>
[2025-07-30T03:12:22.765+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-30T03:12:22.775+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data_task> on 2025-06-25 00:00:00+00:00
[2025-07-30T03:12:22.782+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=533) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-30T03:12:22.782+0000] {standard_task_runner.py:63} INFO - Started process 535 to run task
[2025-07-30T03:12:22.782+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ppppppppppppppppp', 'load_data_task', 'scheduled__2025-06-25T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/dag_generator_python.py', '--cfg-path', '/tmp/tmpcejsa437']
[2025-07-30T03:12:22.784+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask load_data_task
[2025-07-30T03:12:22.819+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ppppppppppppppppp.load_data_task scheduled__2025-06-25T00:00:00+00:00 [running]> on host ee0266895495
[2025-07-30T03:12:22.880+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_ppppppppppppppppp' AIRFLOW_CTX_TASK_ID='load_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-25T00:00:00+00:00'
[2025-07-30T03:12:22.881+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-30T03:12:22.892+0000] {logging_mixin.py:188} INFO - 測試匯入層
[2025-07-30T03:12:22.892+0000] {logging_mixin.py:188} INFO - load_data_task
[2025-07-30T03:12:22.892+0000] {logging_mixin.py:188} INFO - 333333333333333333333333
[2025-07-30T03:12:22.893+0000] {logging_mixin.py:188} INFO - Loading data from Redis key: ***_data:dag_ppppppppppppppppp
[2025-07-30T03:12:22.919+0000] {logging_mixin.py:188} INFO - Saved output CSV to: /opt/***/data/output/3f2a4b93-b237-4545-bd50-5004d863567c.csv
[2025-07-30T03:12:22.920+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-30T03:12:22.920+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-30T03:12:22.927+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ppppppppppppppppp, task_id=load_data_task, run_id=scheduled__2025-06-25T00:00:00+00:00, execution_date=20250625T000000, start_date=20250730T031222, end_date=20250730T031222
[2025-07-30T03:12:22.955+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-30T03:12:22.967+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-30T03:12:22.968+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
